lib/usd/utils/util.cpp

```diff
@@ -207,4 +189,14 @@ updateInternalReferencesPath(const UsdPrim& oldPrim, const SdfPath& newPath)
     return true;
 }
 
+bool
+isInternalReference(const SdfReference& ref)
+{
+    #if USD_VERSION_NUM >= 2008
+    return ref.IsInternal();
+    #else
+    return ref.GetAssetPath().empty();
+    #endif
+}
```

Comment:
IsInternal() is a new routine that I added to SdfReference as part of a PR and is supported as of `20.08`.

https://github.com/PixarAnimationStudios/USD/blob/71b4baace2044ea4400ba802e91667f9ebe342f0/pxr/usd/sdf/reference.h#L155

Added a guard to make sure I am not breaking older USD version ( e.g 19.11 )